scripts/simulate_trade.py

```diff
@@ -221,13 +230,17 @@ def run_model(model, dataloader, factors=None):
             shift = factors.get(model.y_key).get('min')
             data[model.y_key] = data[model.y_key] * scale + shift
 
-        balance, balance_in_time = trade(data, time_key, timstamps, targets, preds, 
+        balance, balance_in_time, total_trades = trade(data, time_key, timstamps, targets, preds, 
                                          balance=args.balance, mode=args.trade_mode, 
                                          risk=args.risk, y_key=model.y_key)
         # raise ValueError(max_drawdown(balance_in_time))
 
         print(f'{conf} -- Final balance: {round(balance, 2)}')
         print(f'{conf} -- Maximum Draw Down : {round(max_drawdown(balance_in_time) * 100, 2)}')
+        print(f'{conf} -- Total gain in %: {round((balance - args.balance) / args.balance * 100, 2)}')
```

Comment:
add some additional logging for easier understanding of how well the model performs